reana/reana_dev/cluster.py

```diff
@@ -71,8 +75,15 @@ def cluster_commands():
     is_flag=True,
     help="Disable default CNI and use e.g. Calico.",
 )
+@click.option(
+    "--kind-version",
```

Comment:
```suggestion
    "--kind-node-version",
```
To make it clear that it's not the version of kind itself, but it's the one used for the node image?